Networking/WebSockets/BuiltInWebSocket.cc

```diff
@@ -252,7 +254,22 @@ namespace litecore::websocket {
                                             "Missing TLS client cert in C4Replicator config"_sl));
                 return false;
             }
-            if ( slice keyData = auth[kC4ReplicatorAuthClientCertKey].asData(); keyData ) {
+            if ( bool isExternal = auth[kC4ReplicatorAuthClientCertKeyIsExternal].asBool(); isExternal ) {
+#ifdef COUCHBASE_ENTERPRISE
```

Comment:
I think it's not important to check for COUCHBSE_ENTERPRISE here as the external key feature itself is not an enterprise feature even though URLEndpointListener and TLSIdentity public API are enterprise only feature?

Reply:
It's engaged by the client cert. I can take the #ifdef out.